src/cdkCommands.test.js

```diff
@@ -13,9 +13,25 @@ describe('cdkCommands', () => {
             AWS_ROLE_NAME: 'role_name',
             AWS_STACK_NAME: 'stack_name',
             INFRASTRUCTURE_PATH: 'some/path/',
+            SKIP_BOOTSTRAP: 'false',
         });
 
         expect(execa.mock.calls[0]).toMatchSnapshot();
         expect(execa.mock.calls[1]).toMatchSnapshot();
     });
+
+    it('should not call bootstrap when SKIP_BOOTSTRAP is not false', async () => {
+        process.chdir = jest.fn();
+
+        await runCdkCommands({
+            AWS_ACCOUNT_ID: 'account_id',
+            AWS_REGION: 'eu-west-1',
+            AWS_ROLE_NAME: 'role_name',
+            AWS_STACK_NAME: 'stack_name',
+            INFRASTRUCTURE_PATH: 'some/path/',
+            SKIP_BOOTSTRAP: 'true',
+        });
+
+        expect(execa.mock.calls[2]).toMatchSnapshot();
```

Comment:
So for this additional test, we need to take into account that `jest` keeps track of the mock calls unless we explicitly clear then in a [hook](https://jestjs.io/docs/en/setup-teardown#repeating-setup-for-many-tests).
```suggestion
   beforeEach(() => jest.clearAllMocks()) // on line 7
```

```suggestion
    expect(execa).toHaveBeenCalledTimes(1);
	expect(execa.mock.calls[0]).toMatchSnapshot();
```

Reply:
Thanks, that's much better!

Reply:
Doh, sorry I missed that @natseg